edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Amqp/Templates.cs

```diff
@@ -30,11 +25,6 @@ public static class Templates
         public static readonly UriPathTemplate ModuleEventsTemplate = new UriPathTemplate(ModuleTelemetryStreamUriFormat.FormatInvariant("{" + DeviceIdTemplateParameterName + "}", "{" + ModuleIdTemplateParameterName + "}"));
         public static readonly UriPathTemplate DeviceFromDeviceBoundTemplate = new UriPathTemplate(DeviceC2DStreamUriFormat.FormatInvariant("{" + DeviceIdTemplateParameterName + "}"));
         public static readonly UriPathTemplate ModuleFromDeviceBoundTemplate = new UriPathTemplate(ModuleC2DStreamUriFormat.FormatInvariant("{" + DeviceIdTemplateParameterName + "}", "{" + ModuleIdTemplateParameterName + "}"));
-        public static readonly UriPathTemplate ServiceToDeviceBoundTemplate = new UriPathTemplate("/messages/deviceBound");
```

Comment:
Are these removed because they are not used?

Reply:
Yes, correct.